package.json

```diff
@@ -28,11 +28,17 @@
     "lint": "tsc --noEmit && eslint 'src/**/*.{js,ts,tsx}' --quiet --fix",
```

Comment:
Do we need / want to modify this "lint" to match the effect of "lint-staged" or if there's a way to sync/link them?  
essentially:
```json
"lint-staged": {
    "**/*.{js,ts,tsx}": [
      "npm run lint",
      "npm run test"
    ]
  },
```

I'd love to have the same lint rules applied across the project: manual or pre-commit. ❤️

Reply:
@mikechu-optimizely, Initially I did that. But for some reason it did not work. Now its working. I might have missed something. Anyways pushed the change. Do let me know if it looks ok now.